pages/installation/docker.rst

```diff
@@ -399,67 +402,40 @@ Start all services with exposed data directories::
 
 Plugins
 =======
+If you want to add plugins, you can put them into a local directory and mount that directory as a volume. This way, you don’t have to create a new docker image every time you want to add a new plugin (or remove an old one).
 
-In order to add plugins you can build a new image based on the existing `graylog/graylog`_ Docker image with the needed plugin included or you add a volume that points to the locally downloaded plugin file.
+Simply create a ``plugin`` folder, download the plugin(s) you want to install into it and mount the directory as a volume into the docker container::
 
-New Docker image
-----------------
+    $ mkdir plugin
+    $ wget https://downloads.graylog.org/releases/graylog-integrations/graylog-integrations-plugins-4.0.2.tgz
+    $ tar -xvzf graylog-integrations-plugins-4.0.2.tgz graylog-integrations-plugins-4.0.2/plugin/graylog-plugin-integrations-4.0.2.jar --directory plugin
+    $ docker run --name graylog --link mongo --link elasticsearch \
+        -p 9000:9000 -p 12201:12201 -p 1514:1514 \
+        -e GRAYLOG_HTTP_EXTERNAL_URI="http://127.0.0.1:9000/" \
+        -v <PATH_TO_LOCAL_PLUGIN_DIR>:/usr/share/graylog/plugin \
```

Comment:
Since we are creating the plugin directory in this example, I think we can also use it in the docker command:

```suggestion
        -v ./plugin:/usr/share/graylog/plugin \
```